lib/createlocaltracks.ts

```diff
@@ -77,11 +81,11 @@ let createLocalTrackCalls = 0;
  * });
  *
  */
-function createLocalTracks(options) {
+export async function createLocalTracks(options?: CreateLocalTracksOptions): Promise<LocalTrack[]> {
   const isAudioVideoAbsent =
     !(options && ('audio' in options || 'video' in options));
 
-  options = Object.assign({
```

Comment:
just curious - why was this change necessary? Did typescript not allow:   
`options  = Object.assign({...}, options)`

Reply:
looks like rest of the changes are caused by options => fullOptions. They can be avoided if we can keep the variable same.

Reply:
I think it's because `options` is of type `CreateLocalTracksOptions` and it's public. The properties we're merging into fullOptions doesn't exists on `CreateLocalTracksOptions` and will cause errors if we keep the variable name. We're essentially creating a new type here since we're adding more properties. Maybe we should define it? Not sure since it's only use here.